package.json

```
@@ -1,13 +1,12 @@
 {
 	"name": "pandaprotector",
-	"version": "3.0.3",
+	"version": "3.0.4",
 	"description": "RSA - Utility Bot",
 	"main": "index.js",
 	"scripts": {
 		"lint": "npx eslint . && npx prettier --check .",
 		"build": "npx tsc",
 		"start": "node dist/index.js",
-		"test": "npm run build && node dist/index.js config-test.json .env.test",
 		"check": "npm run build && npm run lint"
```

For testing purposes. :trollface: 

```suggestion
		"test": "npm run build && npm start",
		"check": "npm run build && npm run lint"
```